content/applications/services/helpdesk/overview/getting_started.rst

```diff
@@ -2,63 +2,180 @@
+SMS text message sent to the customer.
+
+.. important::
+   SMS Text Messaging is an In-App Purchase (IAP) service that requires prepaid credits to work.
+   Refer to :doc:`SMS Pricing and FAQ
+   </applications/marketing/sms_marketing/pricing/pricing_and_faq>`
+   for additional information.
+
+
+To select an existing email template, select it from the :guilabel:`Email Template` field. Click on
+the arrow key to the right of the field to edit the template.
+
+To create a new template, click the field and begin typing a new template title. Then select
+:guilabel:`Create and edit`.
+
+Follow the same steps to select, edit, or create an :guilabel:`SMS Template`.
+
+.. image:: getting_started/sms-template.png
+   :align: center
+   :alt: View of an SMS template setup page in Odoo Helpdesk
+
+
+Assigning stages to a team
+--------------------------
+
+Make a selection in the :guilabel:`Teams` field. More than one team may be selected, as stages
+can be shared among multiple teams.
+
+ .. image:: getting_started/stages-settings-sharing.png
+   :align: center
+   :alt: View of stage setup emphasizing teams field
+
+.. example::
+   In the setup below, the stages `New`, `In Progress`, and `Canceled` are shared between the
+   `Customer Care` and `VIP Support` teams. The other teams are only available to one of the two
+   teams.
+
+  .. image:: getting_started/stages-sharing.png
+   :align: center
+   :alt: View of stages list in Helpdesk showing shared stages
+
+Fold a stage
+------------
+
+Check the :guilabel:`Folded in Kanban` box to display this stage as *folded* by default in the
+kanban view for this team.
 
-To set up stages, go to :menuselection:`Helpdesk --> Configuration --> Stages`. Then, create and/or
-edit stages as you need and set specific teams to use certain stages under *Team*.
+.. warning::
+   Tickets that reach a *folded* stage are considered closed. Closing a ticket before the work is
+   completed can result in reporting and communication issues. This setting should only be enabled
+   for stages that are considered *closing* stages.
 
-.. image:: getting_started/stages_teams.png
+.. image:: getting_started/folded-closing.png
    :align: center
-   :alt: View of a stage’s setting page emphasizing the option to add teams in Odoo Helpdesk
+   :alt: View of the 'Folded in Kanban' setting enabled
 
-Stages can be shared between one or multiple teams, allowing you to adapt the pipeline to your
-individual needs. They also apply a visibility and access rule, as other teams are not able to see
-or use the stage.
+.. tip::
+   Stages can be temporarily folded in the kanban view, as seen below. This will not close the
+   tickets in this stage.
 
-.. image:: getting_started/helpdesk_kanbanview.png
+.. image:: getting_started/folded-in-pipeline.png
    :align: center
-   :alt: View of a team’s kanban view in Odoo Helpdesk
+   :alt: View of Fold setting in Kanban pipeline
```

Comment:
Reduce unnecessary scrolling for the reader by:
- condensing similar information together
- being specific about location and instructions
- removing unnecessary images

```suggestion
Check the :guilabel:`Folded in Kanban` box on the :guilabel:`Stages` form to display this stage as
*folded* by default in the kanban view for this team.

.. warning::
   Tickets that reach a *folded* stage are considered closed. Closing a ticket before the work is
   completed can result in reporting and communication issues. This setting should only be enabled
   for stages that are considered *closing* stages.

Alternatively, stages can be temporarily folded in the kanban view, by clicking on the settings icon
and selecting :guilabel:`Fold`.

.. note::
   Manually folding the stage from the kanban view will not close the tickets in this stage.
```